uber/site_sections/preregistration.py

```diff
@@ -191,7 +191,8 @@ def form(self, session, message='', edit_id=None, **params):
                     if group.badges:
                         Tracking.track(track_type, group)
 
```

Comment:
This is a pattern which you're doing in a bunch of places.  As such, I'd recommend adding a method to the ``SessionMixin`` class, something like

```python
def valid_attendees(self):
    return self.query(Attendee).filter(Attendee.badge_status != c.INVALID_STATUS)
```

Which in turn would allow you to write code like

```python
if session.valid_attendees().filter_by(...
```

which is nicer than having to repeat a block of code (even if it's only part of a line) in a ton of places.